wallet/src/de/schildbach/wallet/ui/CheckPinDialog.kt

```diff
@@ -67,6 +67,8 @@ open class CheckPinDialog : DialogFragment() {
     protected var fingerprintHelper: FingerprintHelper? = null
     protected lateinit var fingerprintCancellationSignal: CancellationSignal
 
+    private var pinLength = PinPreviewView.DEFAULT_PIN_LENGTH
```

Comment:
Why is this var initialized here with `PinPreviewView.DEFAULT_PIN_LENGTH` as value if it's changed to `WalletApplication.getInstance().configuration.pinLength` on the `onCreateDialog` method?

Am I missing something?

Reply:
Fixed